src/InfoViz/Native/FieldSelector/index.js

```diff
@@ -71,8 +72,8 @@ function fieldSelector(publicAPI, model) {
     function renderField(d, i) {
       const field = model.provider.getField(d);
       const fieldContainer = d3.select(this);
-      let legendCell = fieldContainer.select('.legend');
-      let fieldCell = fieldContainer.select('.field');
+      let legendCell = fieldContainer.select(`.${style.legend}`);
+      let fieldCell = fieldContainer.select(`.${style.fieldName}`);
```

Comment:
Hi Aron,

Sounds good but let's get together (hangout) at some point as I'm still catching up on mails. (You can call me directly when that works for you)

Thanks,

Seb